controllers/ibmpowervscluster_controller_test.go

```diff
@@ -1146,16 +1146,161 @@ func TestReconcileVPCResources(t *testing.T) {
 	}
 }
 
-func getServiceInstanceReadyCondition() capiv1beta1.Condition {
-	return capiv1beta1.Condition{
-		Type:   infrav1beta2.ServiceInstanceReadyCondition,
-		Status: "True",
+func TestReconcilePowerVSResources(t *testing.T) {
+	testCases := []struct {
+		name                    string
+		powerVSClusterScopeFunc func() *scope.PowerVSClusterScope
+		reconcileResult         reconcileResult
+		conditions              capiv1beta1.Conditions
+	}{
+		{
+			name: "when Reconciling PowerVS service instance returns error",
```

Comment:
May be for being consistent with other files, lets start with Capital letters for Name!